src/uct/ib/base/ib_iface.c

```diff
@@ -261,8 +261,9 @@ ucs_status_t uct_ib_iface_create_ah(uct_ib_iface_t *iface,
     if (ah == NULL) {
         p    = buf;
         endp = buf + sizeof(buf);
-        snprintf(p, endp - p, "dlid=%d sl=%d port=%d src_path_bits=%d",
-                 ah_attr->dlid, ah_attr->sl, ah_attr->port_num, ah_attr->src_path_bits);
+        snprintf(p, endp - p, "dlid=%d sl=%d traffic_class=%d port=%d src_path_bits=%d",
```

Comment:
need to print trafic_class only if is_global

Reply:
fixed